### PR TITLE
Docs - add docs for .AsServiceBusMessage and other extensions

### LINKED ISSUE #22: Provide Azure Service Bus plugin for adding correlation info to messages
Provide Azure Service Bus plugin for adding correlation info to outbound messages based on the current `ICorrelationInfo`.

For an example of what a Service Bus plugin is, see [SeanFeldman/ServiceBus.AttachmentPlugin](https://github.com/SeanFeldman/ServiceBus.AttachmentPlugin).

### PR DESCRIPTION
**Pull request by stijnmoreels**
Let's keep the builder approach on hold, and continue with documenting the extensions/correlation, so we're done with this milestone.

Closes #22

### CODE REVIEW

File: docs/features/service-bus.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: "Azure Service Bus Features"
+layout: default
+---
+
```
Comment (tomkerkhove) [inline docs/features/service-bus.md]: Can we add installation section similar to this please?

https://github.com/arcus-azure/arcus.messaging/blob/master/docs/features/tcp-health-probe.md#L10

Comment (stijnmoreels) [reply docs/features/service-bus.md]: 👍

File: docs/features/service-bus.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: "Azure Service Bus Features"
```
Comment (tomkerkhove) [inline docs/features/service-bus.md]:
```suggestion
title: "Azure Service Bus Extensions"
```

File: docs/features/service-bus.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+layout: default
+---
+
+# Azure Service Bus Features
+
+The `Arcus.Messaging.ServiceBus.Core` library contains several additional features related to message creation and message/context discoverability.
```
Comment (tomkerkhove) [inline docs/features/service-bus.md]:
```suggestion
We provide several additional features related to message creation and message/context discoverability.
```

**Pull request merged**